addons/rose/addon/components/rose/table/index.stories.mdx

```diff
@@ -60,6 +87,76 @@ A table component to display data in structure of rows and columns. It supports
       </table.footer>
     </Rose::Table>
     `,
-    context: {}
+    context: {
+      hideHeaderRow: boolean('hideHeaderRow', false),
+      cellStyle: select(
+        'cellStyle',
+        {
+          default: '',
+          'expand': 'expand'
+        }
+      ),
+      style: select(
+        'style',
+        {
+          'default': '',
+          'condensed': 'condensed'
+        }
+      ),
+      cellAlignmentCenter: boolean('cellAlignmentCenter', false),
+      cellAlignmentRight: boolean('cellAlignmentRight', false),
+    },
+  }}</Story>
+</Preview>
+
+<Preview>
+  <Story name="Basic Key/Value Table">{{
+    template: hbs`
+      <Rose::Table @style="condensed" as |table|>
+        <table.header as |header|>
+          <header.row @hidden=true as |row|>
+            <row.headerCell>Key</row.headerCell>
+            <row.headerCell @style="expand">Value</row.headerCell>
```

Comment:
This doesn't really feel like a "style".  It could be a boolean.  In either case, I think the need is more for a "shrink" concept.  `@shrink={{true}}`

Reply:
Could we use a `@expand={{true}}` and reuse existing styles? Mainly coz it's easier to apply a width of 100% cell rather than figuring out how small a cell (including it's header) can possibly be.